src/app/components/flyout/flyout-guidelines.component.html

```diff
@@ -0,0 +1,273 @@
+			<stache-row>
+    		<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-scroll-1.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-scroll-2.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-scroll-3.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+		<div class="sky-subsection-heading">Drag to resize</div>
+			<p>Users can select the flyout's left border and drag it to adjust the size. The size can range from 320px to 20px less than the full screen size. If users expand the flyout to the full screen size, it slides 20px to the right to keep a portion of the background page in view. The default size is 50 percent of the screen or 320px, whichever is larger. When users resize the panel, the system remembers their preferred size globally.</p>
+			<stache-row>
+    		<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-resize-1.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-resize-2.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-resize-3.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+		<div class="sky-subsection-heading">Help panel</div>
+			<p>The flyout covers the omnibar, but the help panel remains visible in the upper right corner of the screen. The help panel should recognize that the flyout is open and provide appropriate guidance based on the context.</p>
+			<stache-row>
+    		<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-help-1.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-help-2.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+		<div class="sky-subsection-heading">Modal dialog behavior</div>
+			<stache-row>
+				<stache-column screenSmall="6">
+					<p>When users perform actions in the flyout to launch modal dialogs, the modal dialogs should cover the entire page.</p>
```

Comment:
This says modal dialogs should cover "the entire page," but the image that accompanies this item shows a modal that only covers part of the page. Which is correct?